src/app.module.ts

```diff
@@ -19,7 +20,7 @@ import { ConversationModule } from './conversation/conversation.module';
       host: config.dbHost,
       port: 3306,
       username: config.dbUser,
-      password: config.dbPassword,
+     // password: config.dbPassword,
```

Comment:
Prosiłbym nie wysyłać w commitach zmian ustawień globalnych aplikacji. Jeżeli jest to wymagane to zaleca się to robić w swoim lokalnym pliku config

Reply:
Przepraszam, miałam to odkomentować przed commitem. Pozmieniam u siebei tak żeby już nie ingerować w app'a